sncrs/data/templates/data/teams.html

```diff
@@ -1,5 +1,10 @@
-<table class="table table-borderless table-dark">
-  <tr>
+<table class="table table-borderless table-dark text-center">
```

Comment:
Do these html updates have to do with the containerization at all, or are these just unrelated aesthetic updates made on the side?

Reply:
Great question! The html updates are not related to the containerization, except in that I noticed that without any data in the database, the home page looked really odd. So this was to solve that issue for fresh installs!